src/main/java/jalasoft/com/models/ToDoListPersister.java

```diff
@@ -0,0 +1,53 @@
+package jalasoft.com.models;
+
+import java.util.List;
+import java.util.UUID;
+
+public class ToDoListPersister {
+

```

Comment:
why all the methods of this class are static? if it is an utility class,you should prevent the creation of objects of this class